frost-coordinator/src/coordinator.rs

```diff
@@ -166,22 +166,27 @@ where
         };
 
         let nonce_request_message = Message {
-            sig: nonce_request.sign(&self.network_private_key).expect(""),
+            sig: nonce_request
+                .sign(&self.network_private_key)
+                .expect("Failed to sign NonceRequest"),
             msg: MessageTypes::NonceRequest(nonce_request),
         };
 
-        debug!("dkg_id #{}. NonceRequest sent.", self.current_dkg_id);
+        info!(
+            "dkg_id #{} sign_id #{} sign_nonce_id #{}. NonceRequest sent",
+            self.current_dkg_id, self.current_sign_id, self.current_sign_nonce_id
+        );
```

Comment:
Let's remember to change this back to `debug` once we have proper env logging
```suggestion
        // TODO(#286): Change back to debug level
        info!(
            "dkg_id #{} sign_id #{} sign_nonce_id #{}. NonceRequest sent",
            self.current_dkg_id, self.current_sign_id, self.current_sign_nonce_id
        );
```

Reply:
Done.